packages/core/src/routes/session/session.ts

```diff
@@ -130,6 +140,7 @@ export default function sessionRoutes<T extends AnonymousRouter>(router: T, prov
         username,
         passwordEncrypted,
         passwordEncryptionMethod,
+        roleNames: createAdminUser ? [UserRole.Admin] : [],
```

Comment:
maybe we'd better log that we created an admin user?

```ts
ctx.log(type, { userId: id, roleNames });
```

Reply:
updated